src/compiler/commandLineParser.ts

```diff
@@ -1168,6 +1168,14 @@ namespace ts {
             description: Diagnostics.Emit_ECMAScript_standard_compliant_class_fields,
             defaultValueDescription: "false"
         },
+        {
+            name: "preserveValueImports",
+            type: "boolean",
+            affectsEmit: true,
+            category: Diagnostics.Emit,
+            description: Diagnostics.Disable_the_removal_of_unused_imported_identifiers_from_the_JavaScript_output

```

Comment:
The wording of the description seems confusing. Perhaps something similar to the following:

```
Preserve unused imported values in the JavaScript output that would otherwise be removed.
```